docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,288 @@
+---
+sidebar_position: 2
+description: 'General Information on EDC Policies'
+title: Working with Policies
+id: connector_kit_adoption_view_policies
+---
+
+## Working with Policies
+
+Policies are a formalization of the terms under which a Provider extends an offer to the Dataspace. The format is given
+by the Dataspace Protocol Specification: The [Open Digital Rights Language (ODRL)](https://www.w3.org/TR/odrl-model/).
+
+Like all payloads that get exchanged between connectors, ODRL is an RDF-based description language that is on exchange
```

Comment:
The data space protocol newer talks about "data exchange". It always uses the term "data sharing". Therefore, I would replace every "data exchange" with "data sharing".

Reply:
I think it's irrelevant - both are used interchangeably in the ecosystem. Will replace it here.